ts/models/messageType.ts

```diff
@@ -109,13 +114,33 @@ export interface DataExtractionNotificationMsg {
   referencedAttachmentTimestamp: number; // the attachment timestamp he screenshot
 }
 
+export interface MessageRequestResponseMsg {
+  source: string;
+  isApproved: boolean;
+}
+
+export enum MessageDirection {
+  outgoing = 'outgoing',
+  incoming = 'incoming',
+}
```

Comment:
we already have the types for this. Search for `"outgoing" | "incoming"` and use that typescript type instead everywhere where you are using MessageDirection